src/issue/screens/issue.screen.js

```diff
@@ -240,6 +266,8 @@ class Issue extends Component {
       <CommentListItem
         comment={item}
         onLinkPress={node => this.onLinkPress(node)}
+        onDeletePress={comment => this.deleteComment(comment)}
+        onEditPress={comment => this.editComment(comment)}
```

Comment:
I think we can write shorter as `onDeletePress={this.deleteComment}`. Maybe another pr is required if you guys(@lex111, @jouderianjr) agree with it.

Of course, there are so many places like this. And it doesn't matter.

Reply:
Yeah, I agree with you, I do prefer the shorter version, I just tried to follow the file pattern, but I also agree in do another PR for that, should be better change all the file.

Reply:
Yeah I've used arrow syntax within render callbacks throughout the app. Leveraging `onDeletePress={this.deleteComment}` is better and I think I'll dedicate some time to do that throughout the app